src/nspawn/nspawn-settings.h

```diff
@@ -156,6 +156,7 @@ typedef struct Settings {
         char **environment;
         char *user;
         uint64_t capability;
+        uint64_t ambient_capability;
         uint64_t drop_capability;
```

Comment:
`capability` and `drop_capability` both affect the same set of caps, while `ambient_capability` affects a different set. hence i'd swap the order here, and put ambient_capability last